ui/narrative/methods/run_megahit/display.yaml

```diff
@@ -38,33 +37,30 @@ parameters :
         short-hint : Select a MEGAHIT parameter preset
         description : |
             This option overrides a group of parameters.
-            meta            '--min-count 2 --k-list 21,41,61,81,99'
-            meta-sensitive  '--min-count 2 --k-list 21,31,41,51,61,71,81,91,99'
-            meta-large      '--min-count 2 --k-list 27,37,47,57,67,77,87'
-            bulk            '--min-count 3 --k-list 31,51,71,91,99 --no-mercy'
-            single-cell     '--min-count 3 --k-list 21,33,55,77,99,121 --merge_level 20,0.96'
```

Comment:
It's a great thing we have tests!  yeah- the latest version of megahit has less preset options, and the preset flags that remain actually indicate different parameter values.